include/SFML/Audio/SoundFileFactory.hpp

```diff
@@ -95,7 +96,7 @@ class SFML_AUDIO_API SoundFileFactory
     /// \see createReaderFromMemory, createReaderFromStream
     ///
     ////////////////////////////////////////////////////////////
-    static SoundFileReader* createReaderFromFilename(const std::string& filename);
+    static std::unique_ptr<SoundFileReader> createReaderFromFilename(const std::string& filename);
```

Comment:
In the documentation for this method and the following ones, this sentence should be removed:
> It's up to the caller to release the returned reader

Reply:
Good catch, fixing...